language/stdlib/modules/0L/Reconfigure.move

```diff
@@ -98,5 +109,22 @@ module Reconfigure {
         AutoPay::reconfig_reset_tick(vm);
         Epoch::reset_timer(vm, height_now);
     }
+
+    /// OL function to update withdrawal limits in all validator accounts
+    fun update_validator_withdrawal_limit(vm: &signer) {
+        let validator_set = LibraSystem::get_val_set_addr();
+        let k = 0;
+        while(k < Vector::length(&validator_set)){
+            let addr = *Vector::borrow<address>(&validator_set, k);
+
+            // Check if limits definition is published
+            if(AccountLimits::has_limits_published<GAS>(addr)) {
+                AccountLimits::update_limits_definition<GAS>(vm, addr, 0, LibraConfig::get_epoch_transfer_limit(), 0, 0);
```

Comment:
update limits is used only when limits is already published. hence adding it here.

Reply:
ok. Should it create the struct/publish in the else {} ?

Reply:
So there is no way to create the struct. Since it has to be done on the account. We wont have the account signer here. The other part to add the struct is in the LibraAccount.